examples/tests/sdk_tests/bucket/events.w

```diff
@@ -1,60 +1,92 @@
 bring cloud;
 
-let b = new cloud.Bucket() as "b";
-let counter = new cloud.Counter();
+let b = new cloud.Bucket();
+let idsCounter = new cloud.Counter();
+let table = new cloud.Table(cloud.TableProps {
+  name: "key-history",
+  primaryKey: "_id",
+  columns: {
+    _id: cloud.ColumnType.STRING,
+    key: cloud.ColumnType.STRING,
+    operation: cloud.ColumnType.STRING,
+    source: cloud.ColumnType.NUMBER,
+    }
+  });
+
+  class Util {
+    extern "../external/sleep.js" static inflight sleep(milli: num);
+  }
+
+// the source of onDelete, onUpdate, onCreate are marked as: 1, and for onEvent it's marked as source 2.
```

Comment:
Maybe we can store the value as a string? I think it would make the test more readable